pkgs/development/python-modules/mlx/default.nix

```diff
@@ -25,29 +28,31 @@ let
     rev = "v3.11.3";
     hash = "sha256-7F0Jon+1oWL7uqet5i1IgHX0fUw/+z0QwEcA3zs5xHg=";
   };
+  fmt = fetchFromGitHub {
+    owner = "fmtlib";
+    repo = "fmt";
+    rev = "10.2.1";
+    hash = "sha256-pEltGLAHLZ3xypD/Ur4dWPWJ9BGVXwqQyKcDWVmC3co=";
+  };
 in
 buildPythonPackage rec {
   pname = "mlx";
-  version = "0.18.0";
+  version = "0.21.1";
```

Comment:
Why not using the standard commit conventions for this upgrade? Please also use the following prefix:

```
python312Packages.mlx
```

And not `python3Packages`, as that doesn't trigger builds by oforg.

Reply:
@doronbehar AFAIK ofborg does not use the commit message to determine the rebuilds, but it calculates the rebuild based on the diff in the nix code (correct me if I am wrong).

Reply:
> @doronbehar AFAIK ofborg does not use the commit message to determine the rebuilds, but it calculates the rebuild based on the diff in the nix code (correct me if I am wrong).

Nowadays GitHub actions are responsible for detecting the changed packages of a PR by examining the output paths of all packages with and without the PR. However ofborg is also capable of detecting packages that are directly changed by a package, according to the commits' messages' prefixes. See [this](https://github.com/NixOS/ofborg?tab=readme-ov-file#automatic-building). Hopefully, GitHub actions based CI will do that as well, see:

- https://github.com/NixOS/nixpkgs/issues/355847